.eslintrc

```diff
@@ -55,13 +55,13 @@
     "camelcase": ["off", {"properties": "never"}], // This causes errors with mixin properties
     "comma-dangle": ["error", "never"],
     "compat/compat": "error",
+    "destructuring/in-methods-params": "off",
     "destructuring/in-params": "off",
     "destructuring/no-rename": "off",
     "eqeqeq": ["error", "smart"],
     "jsx-max-len/jsx-max-len": ["error", {"lineMaxLength": 80, "tabWidth": 2, "maxAttributesPerLine": 1}],
     "keyword-spacing": ["error"],
     "multiline-ternary": "off", // allow both multiline and single line
-    "newline-before-return": "error",
```

Comment:
oh, i should have put this into an own commit.

it's because that lint bugs me every single day and we allow a return without a newline in front of it in typescript. will put that up for discussion tomorrow.